crc32.c

```diff
@@ -201,6 +201,7 @@ ZLIB_INTERNAL void crc_finalize(deflate_state *const s) {
 
 ZLIB_INTERNAL void crc_reset(deflate_state *const s) {
 #ifdef X86_PCLMULQDQ_CRC
+    x86_check_features();
```

Comment:
Is this a bugfix or a new requirement?

Reply:
It is a bug fix related to #527. Previously the CPU checks would only get called if `inflate()` or `deflate()` was called. But the `checksum_fuzzer` doesn't use either function and instead uses the crc functions directly, resulting in the CPU check flag variables being optimized out.